draft-ietf-httpbis-header-structure.md

```diff
@@ -587,7 +587,7 @@ Given an ordered dictionary as input_dictionary (each member having a member_nam
 
 ### Serializing an Item {#ser-item}
 
-Given an item bare_item and parameters item_parameters as input, return an ASCII string suitable for use in a HTTP header value.
+Given an item as bare_item and parameters item_parameters as input, return an ASCII string suitable for use in a HTTP header value.
```

Comment:
I think this was right. Otherwise it would be "Given an item as base_item and parameters as item_parameters as input ..."

Reply:
I tend to agree; that's the style we're using elsewhere.

Reply:
Ahh, I didn't read this sentence correctly.

I read it as "GIven (an item named bare_item) and (parameters named item_parameters) as input to this function, ..."

but it's actually saying "Given an item (which is a tuple made up of bare_item and (parameters named item_parameters)) named input, ..."

The name "input" isn't used in the algorithm, which I think threw my parser off. Maybe the style used in #ser-dict would be appropriate here:

"Given an item (having a bare_item and item_parameters), return an ASCII string..."

Reply:
I've just realised the other algorithms use "as" to mean "named", and don't say "as input". For example, bare item:

> Given an item as input_item, return...

The right fix would probably be:

> Given an item as bare_item and parameters as item_parameters, return...

Reply:
Yeah, "as input" isn't used at all elsewhere.

> Given an item as bare_item and parameters as item_parameters, return...

That would work, except that an item already has parameters, according to `{#item}`, I think.
But then again, `{#ser-bare-item}` also claims to take an item, so maybe it just works.

Reply:
Fixed now.